doc/src/fpga_tools/opaevfio/opaevfio.md

```diff
@@ -0,0 +1,68 @@
+# opaevfio #
+
+## SYNOPSIS ##
+
+`opaevfio [-h] [-i] [-r] [-d DRIVER] [-u USER] [-g GROUP] [-n] [-v] [addr]`
+
+## DESCRIPTION ##
+
+The ```opaevfio``` command enables the binding/unbinding of a PCIe device
+to/from the vfio-pci device driver. See https://kernel.org/doc/Documentation/vfio.txt
+for a description of vfio-pci.
+
+`addr`
```

Comment:
Shouldn't we have an ##OPTIONS## sections?

Reply:
Is there precedent in some of the other .md files we've written.  I don't seem to see it.  Point me towards what you're thinking about.